skia-safe/src/core/typeface.rs

```diff
@@ -97,16 +173,18 @@ impl RCHandle<SkTypeface> {
     // encoding of the path name is. from_data can be used instead.
 
     pub fn from_data(data: &Data, index: usize) {
-        Typeface::from_ptr(
-            unsafe {
-                C_SkTypeface_MakeFromData(data.shared_native(), index.try_into().unwrap())
-            } );
+        Typeface::from_ptr(unsafe {
+            C_SkTypeface_MakeFromData(data.shared_native(), index.try_into().unwrap())
+        });
+    }
+
+    pub fn new_clone(&self, arguments: &FontArguments) -> Option<Typeface> {

```

Comment:
I suggest to rename this to `clone_with_arguments`. I would even prefer just `with_arguments`, but to make the transition from the Skia function `makeClone` a more discoverable, it's probably good to include the `clone` prefix.